Documentation/gettingstarted/local-redirect-policy.rst

```diff
@@ -301,5 +309,147 @@ configuring the CiliumLocalRedirectPolicy.
+``coredns_cache_misses_total`` via curling ``<node-local-dns-pod>:9253/metrics``.
+
+kiam redirect on EKS
+--------------------
+`kiam <https://github.com/uswitch/kiam>`_ agent runs on each node in an EKS
+cluster, and intercepts requests going to the AWS metadata server to fetch
+security credentials for pods.
+
+- In order to only redirect traffic from pods to the kiam agent, and pass
+  traffic from the kiam agent to the AWS metadata server without any redirection,
+  we need the socket lookup functionality in the datapath. This functionality
+  requires v5.1.16, v5.2.0 or more recent Linux kernel. Make sure the kernel
+  version installed on EKS cluster nodes satisfies these requirements.
+
+- Deploy kiam.
+
+  .. tabs::
+
+    .. group-tab:: Quick Start
+
+            Deploy kiam using the example yaml file. The example yaml uses
+            hard coded secrets, certs, etc. You may want to replace these
+            with real values from your cluster configuration.
+
+            .. parsed-literal::
+
+                $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes-local-redirect/kiam.yaml -n kube-system
+
+    .. group-tab:: Manual Configuration
+
+        - Deploy `kiam <https://github.com/uswitch/kiam>`_ using helm charts.
+            .. code-block:: bash
+
+                $ helm repo add uswitch https://uswitch.github.io/kiam-helm-charts/charts/
+                $ helm repo update
+                $ helm template kiam uswitch/kiam > kiam.yaml
+
+        - Edit the kiam yaml file, and remove the "--iptables" argument passed to the ``kiam-agent Daemonset``.
+
+        - Make sure the ``kiam-agent Daemonset`` is run in the ``hostNetwork`` mode.
+
+        - Apply the kiam configuration.
+
+            .. code-block:: bash
+
+                $ kubectl apply -f kiam.yaml -n kube-system
+
+        .. note::
+
+            - The ``addressMatcher`` ip address in the local redirect policy is set to
+              the ip address of the AWS metadata server and the ``toPorts`` port
+              to the default HTTP server port. The ``toPorts`` field under
+              ``redirectBackend`` configuration in the policy is set to the port that
+              the kiam agent listens on. The port is passed as "--port" argument in
+              the ``kiam-agent Daemonset``.
+            - The local redirect policy namespace is set to the namespace
+              in which kiam-agent Daemonset is deployed.
+
+
+- Deploy the local redirect policy to redirect pod traffic to the deployed kiam agent.
+
+    .. parsed-literal::
+
+        $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes-local-redirect/kiam-lrp.yaml
+
+- Once all the kiam agent pods are in ``Running`` state, the metadata requests
+  from application pods will get redirected to the node-local kiam agent pods.
+  You can verify this by running ``cilium monitor`` with ``related-to`` set to
+  the endpoint id of one of the application pods.
```

Comment:
This is still vague, and assumes the user knows what they're verifying. Many users will not even be familiar with `cilium monitor` particularly as Hubble becomes more prevalent. If we're going to use this as an example, I would suggest spelling it out with an example that shows what the output looks like and how to interpret it.

Reply:
done